test/unit/lifecycle/properties.js

```diff
@@ -76,6 +76,41 @@ describe('lifecycle/properties', function () {
     expect(el.getAttribute('my-attr')).to.equal('test2', 'attribute updated to new value');
   });
 
+  describe('attribute - setting', function() {
+    it('with `attr` set to true', function() {
+      skate(elem.safe, {
+        properties: {
+          test: {
+            attr: true
+          }
+        }
+      });
+
+      var el = fixture(`<${elem.safe}></${elem.safe}>`).querySelector(elem.safe);
+      skate.init(el);
+
+      el.test = 'value1';
+      el.setAttribute('test', 'value2');
+      expect(el.test).to.equal('value2');
+    });
```

Comment:
This test passed without any of the changes made, but is added here for better coverage.